tests/components/switchbot/__init__.py

```diff
@@ -16,7 +16,7 @@
 }
 
 USER_INPUT = {
-    CONF_ADDRESS: "aa:bb:cc:dd:ee:ff",
+    CONF_ADDRESS: "AA:BB:CC:DD:EE:FF",
```

Comment:
What was wrong with the old lowercase address? This seems like unnecessary churn.

Reply:
Agree with ucodery. lets keep this out unless it is really necessary